cmd/query/app/querysvc/query_service.go

```diff
@@ -99,16 +111,16 @@ func (qs QueryService) GetOperations(
 }
 
 // FindTraces is the queryService implementation of spanstore.Reader.FindTraces
-func (qs QueryService) FindTraces(ctx context.Context, query *spanstore.TraceQueryParameters) ([]*model.Trace, error) {
+func (qs QueryService) FindTraces(ctx context.Context, query *TraceQueryParameters) ([]*model.Trace, error) {
 	spanReader, err := v1adapter.GetV1Reader(qs.traceReader)
 	if err != nil {
 		return nil, err
 	}
-	return spanReader.FindTraces(ctx, query)
+	return spanReader.FindTraces(ctx, &query.TraceQueryParameters)
 }
 
 // ArchiveTrace is the queryService utility to archive traces.
-func (qs QueryService) ArchiveTrace(ctx context.Context, query spanstore.GetTraceParameters) error {
+func (qs QueryService) ArchiveTrace(ctx context.Context, query GetTraceParameters) error {
```

Comment:
should this remain `spanstore.GetTraceParameters`? Or ArchiveTraceParameters, but Get with `raw` doesn't apply here.

Reply:
@yurishkuro Thanks for catching. This was an oversight - I reverted it back for now. We can always create a separate type here if we need to in the future.